pkg/controller/tidb_control.go

```diff
@@ -63,10 +64,20 @@ func (tdc *defaultTiDBControl) GetHealth(tc *v1alpha1.TidbCluster) map[string]bo
 	tcName := tc.GetName()
 	ns := tc.GetNamespace()
 
+	scheme := "http"
+	if tc.Spec.EnableTLSServer {
+		scheme = "https"
+		rootCAs, _ := httputil.ReadCACerts()
```

Comment:
Don't ignore errors in Go. if you're definitely sure the error would never happen, then add a comment for annotation.